<robot name="iai-kitchen" version="1.0"><link name="shelf:shelf:shelf_origin" />
  <link name="shelf:shelf:shelf_center" />
  <link name="shelf:shelf:shelf_base_center" />
  <link name="shelf:shelf:shelf_back">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.01 1.04 1.355" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.01 1.04 1.355" />
      </geometry>
    </collision>
  </link>
  <link name="shelf:shelf:shelf_right_side">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 0.01 1.355" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 0.01 1.355" />
      </geometry>
    </collision>
  </link>
  <link name="shelf:shelf:shelf_left_side">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 0.01 1.355" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 0.01 1.355" />
      </geometry>
    </collision>
  </link>
  <link name="shelf:shelf:shelf_floor_0">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf:shelf:shelf_floor_1">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf:shelf:shelf_floor_2">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.34 1.04 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="iai_kitchen:kitchen:room_center_link">
    <visual>
      <geometry>
        <box size="5.25 3.0 0.01" />
      </geometry>
      <material name="iai_kitchen:kitchen:color">
        <color rgba="0.0 0.0 1.0 0.1" />
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="5.25 3.0 0.01" />
      </geometry>
    </collision>
  </link>
  <link name="kitchen_entry_exit:entry_exit:link" />
  <link name="arena:arena:room_center_link">
    <visual>
      <geometry>
        <box size="5.2 9.1 0.01" />
      </geometry>
      <material name="arena:arena:color">
        <color rgba="1.0 0.0 0.0 0.0" />
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="5.2 9.1 0.01" />
      </geometry>
    </collision>
  </link>
  <link name="arena_entry:entry:link" />
  <link name="arena_exit:exit:link" />
  <link name="living_room:living_room:room_center_link">
    <visual>
      <geometry>
        <box size="5.25 3.4 0.01" />
      </geometry>
      <material name="living_room:living_room:color">
        <color rgba="0.0 1.0 0.0 0.1" />
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="5.25 3.4 0.01" />
      </geometry>
    </collision>
  </link>
  <link name="living_room_entry_exit:entry_exit:link" />
  <link name="dining_room:dining_room:room_center_link">
    <visual>
      <geometry>
        <box size="5.25 2.7 0.01" />
      </geometry>
      <material name="dining_room:dining_room:color">
        <color rgba="1.0 0.0 0.0 0.1" />
      </material>
    </visual>
    <collision>
      <geometry>
        <box size="5.25 2.7 0.01" />
      </geometry>
    </collision>
  </link>
  <link name="dining_room_entry_exit:entry_exit:link" />
  <link name="iai_kitchen:arena:door_hinge">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.05 0.05 2.0" />
      </geometry>
      <material name="hinge_red" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.05 0.05 2.0" />
      </geometry>
    </collision>
  </link>
  <link name="living_room:arena:door_hinge">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.05 0.05 2.0" />
      </geometry>
      <material name="hinge_red" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.05 0.05 2.0" />
      </geometry>
    </collision>
  </link>
  <link name="dishwasher_main">
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <mass value="1.0" />
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0" />
    </inertial>
  </link>
  <link name="sink_area_dish_washer_main">
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <mass value="1.0" />
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0" />
    </inertial>
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/misc/DishWasher.obj" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/misc/DishWasher.obj" />
      </geometry>
    </collision>
  </link>
  <link name="sink_area_dish_washer_door">
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <mass value="1.0" />
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0" />
    </inertial>
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="1.57079632679 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/misc/DishWasherDoor.obj" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="1.57079632679 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/misc/DishWasherDoor.obj" />
      </geometry>
    </collision>
  </link>
  <link name="sink_area_dish_washer_door_handle">
    <inertial>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <mass value="1.0" />
      <inertia ixx="1.0" ixy="0.0" ixz="0.0" iyy="1.0" iyz="0.0" izz="1.0" />
    </inertial>
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/handles/Handle60.obj" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <mesh filename="package://iai_kitchen/meshes/handles/Handle60.obj" />
      </geometry>
    </collision>
  </link>
  <link name="dishwasher_table:d_table:table_origin" />
  <link name="dishwasher_table:d_table:table_front_edge_center" />
  <link name="dishwasher_table:d_table:table_center">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.6 0.6 0.02" />
      </geometry>
      <material name="dishwasher_table_color" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.6 0.6 0.02" />
      </geometry>
    </collision>
  </link>
  <link name="chair:chair:chair_center">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.3 0.3 0.5" />
      </geometry>
      <material name="chair_color" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.3 0.3 0.5" />
      </geometry>
    </collision>
  </link>
  <link name="chair:chair:chair_backrest">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.3 0.05 0.5" />
      </geometry>
      <material name="chair_color" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.3 0.05 0.5" />
      </geometry>
    </collision>
  </link>
  <link name="chair:chair:chair_origin" />
  <link name="chair:chair:chair_front_edge_center" />
  <link name="shelf_hohc:shelf_hohc:shelf_origin" />
  <link name="shelf_hohc:shelf_hohc:shelf_center" />
  <link name="shelf_hohc:shelf_hohc:shelf_base_center" />
  <link name="shelf_hohc:shelf_hohc:shelf_back">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.8 2.02" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.8 2.02" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_right_side">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.02 2.02" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.02 2.02" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_left_side">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.02 2.02" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.02 2.02" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_floor_0">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_floor_1">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_floor_2">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_floor_3">
    <visual>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
      <material name="shelf_blue" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 -0.02" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.4 0.8 0.04" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_door_right">
    <visual>
      <origin xyz="0.0 0.18 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.38 0.97" />
      </geometry>
      <material name="shelf_door_green" />
    </visual>
    <collision>
      <origin xyz="0.0 0.18 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.38 0.97" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_door_right:handle">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.015 0.015" />
      </geometry>
      <material name="shelf_hohc_handle_red" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.015 0.015" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_door_left">
    <visual>
      <origin xyz="0.0 -0.18 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.38 0.97" />
      </geometry>
      <material name="shelf_door_green" />
    </visual>
    <collision>
      <origin xyz="0.0 -0.18 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.38 0.97" />
      </geometry>
    </collision>
  </link>
  <link name="shelf_hohc:shelf_hohc:shelf_door_left:handle">
    <visual>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.015 0.015" />
      </geometry>
      <material name="shelf_hohc_handle_red" />
    </visual>
    <collision>
      <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
      <geometry>
        <box size="0.02 0.015 0.015" />
      </geometry>
    </collision>
  </link>
  <link name="urdf_main" />
  <link name="urdf_offset" />
<joint name="shelf:shelf:shelf_origin_joint" type="fixed">
    <origin xyz="5.11 4.42 0.0" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="shelf:shelf:shelf_origin" />
  </joint>
  <joint name="shelf:shelf:shelf_center_joint" type="fixed">
    <origin xyz="0.17 0.52 0.6775" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_origin" />
    <child link="shelf:shelf:shelf_center" />
  </joint>
  <joint name="shelf:shelf:shelf_base_center_joint" type="fixed">
    <origin xyz="0.17 0.52 0.0" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_origin" />
    <child link="shelf:shelf:shelf_base_center" />
  </joint>
  <joint name="shelf:shelf:shelf_back:joint" type="fixed">
    <origin xyz="0.34 0.52 0.6775" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_origin" />
    <child link="shelf:shelf:shelf_back" />
  </joint>
  <joint name="shelf:shelf:shelf_right_side:joint" type="fixed">
    <origin xyz="0.17 0.0 0.6775" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_origin" />
    <child link="shelf:shelf:shelf_right_side" />
  </joint>
  <joint name="shelf:shelf:shelf_left_side:joint" type="fixed">
    <origin xyz="0.17 1.04 0.6775" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_origin" />
    <child link="shelf:shelf:shelf_left_side" />
  </joint>
  <joint name="shelf:shelf:shelf_floor_0:joint" type="fixed">
    <origin xyz="0.0 0.0 0.077" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_base_center" />
    <child link="shelf:shelf:shelf_floor_0" />
  </joint>
  <joint name="shelf:shelf:shelf_floor_1:joint" type="fixed">
    <origin xyz="0.0 0.0 0.371" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_base_center" />
    <child link="shelf:shelf:shelf_floor_1" />
  </joint>
  <joint name="shelf:shelf:shelf_floor_2:joint" type="fixed">
    <origin xyz="0.0 0.0 0.716" rpy="0.0 0.0 0.0" />
    <parent link="shelf:shelf:shelf_base_center" />
    <child link="shelf:shelf:shelf_floor_2" />
  </joint>
  <joint name="world_iai_kitchen:kitchen:room_joint" type="fixed">
    <origin xyz="2.825 -1.0 -0.05" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="iai_kitchen:kitchen:room_center_link" />
  </joint>
  <joint name="room_kitchen_entry_exit:entry_exit:connection_joint" type="fixed">
    <origin xyz="-2.225 1.09 0.0" rpy="0.0 0.0 0.0" />
    <parent link="iai_kitchen:kitchen:room_center_link" />
    <child link="kitchen_entry_exit:entry_exit:link" />
  </joint>
  <joint name="world_arena:arena:room_joint" type="fixed">
    <origin xyz="3.0 2.05 -0.05" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="arena:arena:room_center_link" />
  </joint>
  <joint name="room_arena_entry:entry:connection_joint" type="fixed">
    <origin xyz="-2.33 -2.05 0.0" rpy="0.0 0.0 0.0" />
    <parent link="arena:arena:room_center_link" />
    <child link="arena_entry:entry:link" />
  </joint>
  <joint name="room_arena_exit:exit:connection_joint" type="fixed">
    <origin xyz="-2.33 1.75 0.0" rpy="0.0 0.0 0.0" />
    <parent link="arena:arena:room_center_link" />
    <child link="arena_exit:exit:link" />
  </joint>
  <joint name="world_living_room:living_room:room_joint" type="fixed">
    <origin xyz="2.825 2.2 -0.05" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="living_room:living_room:room_center_link" />
  </joint>
  <joint name="room_living_room_entry_exit:entry_exit:connection_joint" type="fixed">
    <origin xyz="-2.5650000000000004 1.6099999999999999 0.0" rpy="0.0 0.0 0.0" />
    <parent link="living_room:living_room:room_center_link" />
    <child link="living_room_entry_exit:entry_exit:link" />
  </joint>
  <joint name="world_dining_room:dining_room:room_joint" type="fixed">
    <origin xyz="2.825 5.25 -0.05" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="dining_room:dining_room:room_center_link" />
  </joint>
  <joint name="room_dining_room_entry_exit:entry_exit:connection_joint" type="fixed">
    <origin xyz="-0.7250000000000001 -1.35 0.0" rpy="0.0 0.0 0.0" />
    <parent link="dining_room:dining_room:room_center_link" />
    <child link="dining_room_entry_exit:entry_exit:link" />
  </joint>
  <joint name="iai_kitchen:arena:door_hinge_joint" type="fixed">
    <origin xyz="0.58 -0.44 1.04" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="iai_kitchen:arena:door_hinge" />
    <limit effort="1.0" lower="0.0" upper="1.5708" velocity="1.0" />
  </joint>
  <joint name="living_room:arena:door_hinge_joint" type="fixed">
    <origin xyz="0.28 3.37 1.04" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="living_room:arena:door_hinge" />
    <limit effort="1.0" lower="0.0" upper="1.5708" velocity="1.0" />
  </joint>
  <joint name="dishwasher_main" type="fixed">
    <origin xyz="1.86 -2.2 0.0" rpy="0.0 0.0 1.57" />
    <parent link="urdf_offset" />
    <child link="dishwasher_main" />
  </joint>
  <joint name="dishwasher_main_joint" type="fixed">
    <origin xyz="1.86 -2.2 0.0" rpy="0.0 0.0 1.57" />
    <parent link="dishwasher_main" />
    <child link="sink_area_dish_washer_main" />
  </joint>
  <joint name="sink_area_dish_washer_main_joint" type="fixed">
    <origin xyz="0.08 -0.78 0.44" rpy="0.0 0.0 0.0" />
    <axis xyz="1.0 0.0 0.0" />
    <parent link="dishwasher_main" />
    <child link="sink_area_dish_washer_main" />
  </joint>
  <joint name="sink_area_dish_washer_door_joint" type="revolute">
    <origin xyz="0.27 0.0 -0.325" rpy="-1.57079632679 0.03 0.0" />
    <axis xyz="0.0 0.0 1.0" />
    <parent link="sink_area_dish_washer_main" />
    <child link="sink_area_dish_washer_door" />
    <limit effort="300.0" lower="0.0" upper="1.57079632679" velocity="10.0" />
  </joint>
  <joint name="sink_area_dish_washer_door_handle_joint" type="fixed">
    <origin xyz="0.0625 -0.6325 0.0" rpy="-1.57079632679 0.0 0.0" />
    <parent link="sink_area_dish_washer_door" />
    <child link="sink_area_dish_washer_door_handle" />
  </joint>
  <joint name="dish_washer_table_main_joint" type="fixed">
    <origin xyz="0.0 0.0 0.39" rpy="0.0 0.0 0.0" />
    <parent link="sink_area_dish_washer_main" />
    <child link="dishwasher_table:d_table:table_origin" />
  </joint>
  <joint name="dish_washer_table_center_joint" type="fixed">
    <origin xyz="0.3 0.0 0.0" rpy="0.0 0.0 0.0" />
    <parent link="dishwasher_table:d_table:table_origin" />
    <child link="dishwasher_table:d_table:table_front_edge_center" />
  </joint>
  <joint name="dish_washer_table_edge_joint" type="fixed">
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
    <parent link="dishwasher_table:d_table:table_origin" />
    <child link="dishwasher_table:d_table:table_center" />
  </joint>
  <joint name="chair:chair:chair_origin_joint" type="fixed">
    <origin xyz="2.8 1.3 0.0" rpy="0.0 0.0 0.0" />
    <parent link="urdf_offset" />
    <child link="chair:chair:chair_origin" />
  </joint>
  <joint name="chair:chair:center_joint" type="fixed">
    <origin xyz="0.0 0.0 0.25" rpy="0.0 0.0 0.0" />
    <parent link="chair:chair:chair_origin" />
    <child link="chair:chair:chair_center" />
  </joint>
  <joint name="chair:chair:chair_front_edge_center_joint" type="fixed">
    <origin xyz="0.0 0.15 0.5" rpy="0.0 0.0 0.0" />
    <parent link="chair:chair:chair_origin" />
    <child link="chair:chair:chair_front_edge_center" />
  </joint>
  <joint name="chair:chair:chair_backrest_joint" type="fixed">
    <origin xyz="0.0 -0.12 0.25" rpy="0.0 0.0 0.0" />
    <parent link="chair:chair:chair_center" />
    <child link="chair:chair:chair_backrest" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_origin_joint" type="fixed">
    <origin xyz="3.4 3.4 0.0" rpy="0.0 0.0 1.57" />
    <parent link="urdf_offset" />
    <child link="shelf_hohc:shelf_hohc:shelf_origin" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_center_joint" type="fixed">
    <origin xyz="0.2 0.4 1.01" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_origin" />
    <child link="shelf_hohc:shelf_hohc:shelf_center" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_base_center_joint" type="fixed">
    <origin xyz="0.2 0.4 0.0" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_origin" />
    <child link="shelf_hohc:shelf_hohc:shelf_base_center" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_back:joint" type="fixed">
    <origin xyz="0.4 0.4 1.01" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_origin" />
    <child link="shelf_hohc:shelf_hohc:shelf_back" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_right_side:joint" type="fixed">
    <origin xyz="0.2 0.01 1.01" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_origin" />
    <child link="shelf_hohc:shelf_hohc:shelf_right_side" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_left_side:joint" type="fixed">
    <origin xyz="0.2 0.79 1.01" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_origin" />
    <child link="shelf_hohc:shelf_hohc:shelf_left_side" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_floor_0:joint" type="fixed">
    <origin xyz="0.0 0.0 0.1" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_base_center" />
    <child link="shelf_hohc:shelf_hohc:shelf_floor_0" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_floor_1:joint" type="fixed">
    <origin xyz="0.0 0.0 0.54" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_base_center" />
    <child link="shelf_hohc:shelf_hohc:shelf_floor_1" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_floor_2:joint" type="fixed">
    <origin xyz="0.0 0.0 1.055" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_base_center" />
    <child link="shelf_hohc:shelf_hohc:shelf_floor_2" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_floor_3:joint" type="fixed">
    <origin xyz="0.0 0.0 2.02" rpy="0.0 0.0 0.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_base_center" />
    <child link="shelf_hohc:shelf_hohc:shelf_floor_3" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_door_right:handle_joint" type="fixed">
    <origin xyz="-0.03 0.3575 0.09249999999999997" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 1.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_door_right" />
    <child link="shelf_hohc:shelf_hohc:shelf_door_right:handle" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_door_right:joint" type="revolute">
    <origin xyz="-0.19 0.02 -0.48" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 1.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_right_side" />
    <child link="shelf_hohc:shelf_hohc:shelf_door_right" />
    <limit effort="30.0" lower="0.0" upper="1.7" velocity="1.0" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_door_left:handle_joint" type="fixed">
    <origin xyz="-0.03 -0.3575 0.09249999999999997" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 1.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_door_left" />
    <child link="shelf_hohc:shelf_hohc:shelf_door_left:handle" />
  </joint>
  <joint name="shelf_hohc:shelf_hohc:shelf_door_left:joint" type="revolute">
    <origin xyz="-0.19 -0.02 -0.48" rpy="0.0 0.0 0.0" />
    <axis xyz="0.0 0.0 1.0" />
    <parent link="shelf_hohc:shelf_hohc:shelf_left_side" />
    <child link="shelf_hohc:shelf_hohc:shelf_door_left" />
    <limit effort="30.0" lower="-1.7" upper="0.0" velocity="1.0" />
  </joint>
  <joint name="map_urdf_joint" type="fixed">
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0" />
    <parent link="urdf_main" />
    <child link="urdf_offset" />
  </joint>
  </robot>